### ISSUE TITLE
teamsView: updated courses not seen on courses tab

### ISSUE THREAD

**Issue by anishregmee** [open-learning-exchange/planet#5186]
>in teams view add a course with description
>then in courses, update that course description
>that updated course description does not display on teamsView courses tab

Solution:
it should work same as resources coz resources tab in teamsView shows the updated resources

**Comment by paulbert** [comment]
Dependent on #5185 so putting on hold

**Comment by paulbert** [comment]
Same as #5637 so closing
